packages/dd-trace/test/opentracing/tracer.spec.js

```diff
@@ -120,6 +120,7 @@ describe('Tracer', () => {
         startTime: fields.startTime,
         hostname: undefined,
         traceId128BitGenerationEnabled: undefined,
+        apmTracingEnabled: undefined,
```

Comment:
Is this possible? It is `true` by default, and when ASM standalone is enabled, it will be `false`, but if I understood it correctly, it can't be `undefined`.

Reply:
It's only for the test, config is a mocked object.
Changed.